plugins/foreman_discovery/8.0/index.md

```diff
@@ -0,0 +1,1302 @@
+      -O - | tar x --overwrite -C /var/lib/tftpboot/boot
+
+Check the version number in the url against the table above. Some image-plugin
+combinations are not compatible.
+
+On Debian systems, use `/srv/tftp/boot` instead of `/var/tftpboot/boot`.
+
+### 2.3.3 Verify signatures and checksums
+
+To verify individual ISO/tar files, import our Discovery Plugin key
+[7E81E7B0](http://keys.fedoraproject.org:11371/pks/lookup?search=0x7E81E7B0&op=get)
+and verify individual files:
+
+    # wget 'http://keys.fedoraproject.org:11371/pks/lookup?search=0x7E81E7B0&op=get' -O- | gpg --import
+    # gpg --verify --multifile *.asc
+    gpg: Signature made Fri 06 Nov 2015 09:42:55 AM UTC using RSA key ID 7E81E7B0
+    gpg: Good signature from "Foreman Discovery <foreman-xxx@googlegroups.com>"
+
+To verify extracted init RAM disk and linux kernel do:
+
+    # cat /var/lib/tftpboot/boot/fdi-image/SHA256SUM
+    beb3cfba7d9fb9d71481c0c8f... initrd0.img
+    f03bce150d2473a28697d2988... vmlinuz0
+    # sha256sum /var/lib/tftpboot/boot/fdi-image/*
+
+On Debian systems, use `/srv/tftp/boot` instead of `/var/tftpboot/boot`.
+
+### 2.3.4 Building own image
+
+To build a discovery image, please visit the foreman-discovery-image git
+[repository](https://github.com/theforeman/foreman-discovery-image) and find
+the README for further instructions.
+
+## 2.4 Hammer CLI Foreman Discovery
+
+### 2.4.1 Installation
+
+Before installing make sure you have installed the
+[hammer_cli_foreman plugin](https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md),
+
+On Red Hat compatible systems:
+
+```yum install tfm-rubygem-hammer_cli_foreman_discovery```
+
+On Debian/Ubuntu compatible systems:
+
+```apt-get install ruby-hammer-cli-foreman-discovery```
+
+Or install from rubygems:
+
+```gem install hammer_cli_foreman_discovery```.
+
+## 2.5 Upgrade
+
+To upgrade Foreman Discovery follow the standard procedure of upgrading all
+the Foreman packages. Check Foreman Discovery global settings and PXELinux
+Default template.
+
+After each upgrade, it is required to reboot all discovered hosts to load new
+discovery image which is compatible with the recent features added in the
+Foreman plugin. To do this, click on Reboot All button in the UI or do this
+via CLI.
+
+# 3. Configuration
+
+The following chapter covers configuration of all the components installed.
+
+## 3.1 Foreman Discovery plugin
+
+Foreman Discovery relies on intercepting the normal boot process for machines
+not registered in Foreman. To achieve this, the PXE *default.cfg* file needs to
+be altered to instruct new machines to boot the discovery image.
+
+## 3.1.1 Default PXE template
+
+In the Foreman UI, go to Provisioning Templates, edit *PXELinux global
+default* template and change the default entry from localboot entry to
+Discovery:
+
+    ONTIMEOUT discovery
+
+The template already contains a menu entry similar to this and nothing needs
```

Comment:
The template actually just contains `<%= snippet "pxelinux_discovery" %>`, best to tell people to take a look there.